models/reporter.go

```diff
@@ -311,128 +399,128 @@ func (r *Reporter) SelectSparklines(ctx context.Context, dimensionVal string,
 `
 const queryServiceType = `
-	SELECT 'service_type' AS key, service_type AS value, SUM(%s) AS main_metric_sum
-	  FROM metrics
-	 WHERE period_start >= ?
-	   AND period_start <= ?
+        SELECT 'service_type' AS key, service_type AS value, SUM(%s) AS main_metric_sum
+          FROM metrics
+         WHERE period_start >= ?
+           AND period_start <= ?
   GROUP BY service_type
       WITH TOTALS
   ORDER BY main_metric_sum DESC, value;
 `
 const queryEnvironment = `
-	SELECT 'environment' AS key, environment AS value, SUM(%s) AS main_metric_sum
-	  FROM metrics
-	 WHERE period_start >= ?
-	   AND period_start <= ?
+        SELECT 'environment' AS key, environment AS value, SUM(%s) AS main_metric_sum
+          FROM metrics
+         WHERE period_start >= ?
+           AND period_start <= ?
   GROUP BY environment
       WITH TOTALS
   ORDER BY main_metric_sum DESC, value;
 `
 const queryAZ = `
-	SELECT 'az' AS key, az AS value, SUM(%s) AS main_metric_sum
-	  FROM metrics
-	 WHERE period_start >= ?
-	   AND period_start <= ?
+        SELECT 'az' AS key, az AS value, SUM(%s) AS main_metric_sum
+          FROM metrics
+         WHERE period_start >= ?
+           AND period_start <= ?
   GROUP BY az
       WITH TOTALS
   ORDER BY main_metric_sum DESC, value;
 `
 const queryRegion = `
-	SELECT 'region' AS key, region AS value, SUM(%s) AS main_metric_sum
-	  FROM metrics
-	 WHERE period_start >= ?
-	   AND period_start <= ?
+        SELECT 'region' AS key, region AS value, SUM(%s) AS main_metric_sum
+          FROM metrics
+         WHERE period_start >= ?
+           AND period_start <= ?
   GROUP BY region
       WITH TOTALS
   ORDER BY main_metric_sum DESC, value;
 `
 const queryNodeModel = `
-	SELECT 'node_model' AS key, node_model AS value, SUM(%s) AS main_metric_sum
-	  FROM metrics
-	 WHERE period_start >= ?
-	   AND period_start <= ?
+        SELECT 'node_model' AS key, node_model AS value, SUM(%s) AS main_metric_sum
+          FROM metrics
+         WHERE period_start >= ?
+           AND period_start <= ?
   GROUP BY node_model
       WITH TOTALS
   ORDER BY main_metric_sum DESC, value;
 `
 const queryContainerName = `
-	SELECT 'container_name' AS key, container_name AS value, SUM(%s) AS main_metric_sum
-	  FROM metrics
-	 WHERE period_start >= ?
-	   AND period_start <= ?
+        SELECT 'container_name' AS key, container_name AS value, SUM(%s) AS main_metric_sum
+          FROM metrics
+         WHERE period_start >= ?
+           AND period_start <= ?
   GROUP BY container_name
       WITH TOTALS
   ORDER BY main_metric_sum DESC, value;
 `
 const queryLabels = `
-	SELECT labels.key AS key, labels.value AS value, SUM(%s) AS main_metric_sum
-	  FROM metrics
+        SELECT labels.key AS key, labels.value AS value, SUM(%s) AS main_metric_sum
+          FROM metrics
 ARRAY JOIN labels
-	 WHERE period_start >= ?
-	   AND period_start <= ?
+         WHERE period_start >= ?
+           AND period_start <= ?
```

Comment:
There are a lot of spaces/tabs changes. Are they intended to make things better, or unintended?

Reply:
Thanks, good question.
Yes, my intent was not to mix tabs and spaces in query formatting.
This simplify reading, copy and execute queries from logs into ClickHouse client.
And also ClickHouse Client displays queries in same way.